.github/workflows/changed_file_trigger_examples_check.yml

```diff
@@ -0,0 +1,128 @@
+name: Test Example
+on:
+  pull_request:
+  schedule:
+    # run at 00:00 of every Sunday(singapore time) so here is UTC time Saturday 16:00
+    - cron:  '0 16 * * 6'
+
+jobs:
+# This is for changed files detect and check.
+  detect-changed-files:
+    runs-on: ubuntu-latest
```

Comment:
`if` is missing to check whether this workflow should run on PRs to the main branch of the org repo if the event name is `pull_request`

Reply:
OK, done!